spells.yml

```diff
@@ -3604,6 +3604,32 @@ fey_mischief:
     - illusion
     - sorcerer
     - wizard
+fiery_whip:
+  casting_time: 1 minor action
+  components: V, S
+  concentration: true
+  description: |-
+    A whip of fire appears in your free hand. The whip is a weapon with the finesse, light, and reach 15 properties. On a hit, the whip deals 2d4 fire damage, and if the target is no more than one size larger than you, you can pull it up to 5 feet towards you.
```

Comment:
How would you feel about letting it drag the creature 10 feet instead of 5? That way, if you had another normal melee weapon in your other hand, you could attack them with it after dragging the creature from 15 feet away.

Reply:
I guess currently, you'd have to hit them twice, and then attack with an offhand weapon. But using the whip as the offhand and making two attacks with another weapon is cool too, and it's hardly a buff since you can almost always just walk 5 feet closer... So sounds good